src/values/fn_value.rs

```diff
@@ -505,6 +507,25 @@ impl<'ctx> FunctionValue<'ctx> {
     pub fn as_global_value(self) -> GlobalValue<'ctx> {
         GlobalValue::new(self.as_value_ref())
     }
+
+    /// Set the debug info descriptor
+    pub fn set_subprogram(self, subprogram: DISubProgram<'ctx>) {
```

Comment:
The two imports and functions calls here need the `#[llvm_versions(6.0..=latest)]` attr applied to play nice with older versions

Reply:
FYI, LLVMSetSubprogram was added in 7.0 not 6.0.